data/kiwi/appliance.kiwi

```diff
@@ -0,0 +1,391 @@
+                <volume name="boot/grub2/i386-pc"/>
+                <volume name="boot/grub2/x86_64-efi" mountpoint="boot/grub2/x86_64-efi"/>
+                <volume name="usr/local"/>
+                <volume name="var" copy_on_write="false"/>
+            </systemdisk>
+            <size unit="G">24</size>
+        </type>
+    </preferences>
+
+    <preferences profiles="RaspberryPi">
+        <version>15.4.0</version>
+        <packagemanager>zypper</packagemanager>
+        <bootsplash-theme>SLE</bootsplash-theme>
+        <bootloader-theme>SLE</bootloader-theme>
+    <!-- those settings are applied by suseConfig in config.sh
+        <locale>en_US</locale>
+        <keytable>us.map.gz</keytable>
+        <timezone>Europe/Berlin</timezone>
+        <hwclock>utc</hwclock>
+    -->
+        <rpm-excludedocs>true</rpm-excludedocs>
+        <type
+            image="oem"
+            initrd_system="dracut"
+            filesystem="btrfs"
+            fsmountoptions="noatime,compress=lzo"
+            bootloader="grub2"
+            firmware="efi"
+            kernelcmdline="rw systemd.show_status=1 console=ttyS0,115200 console=tty quiet"
+            bootpartition="false"
+            devicepersistency="by-uuid"
+            btrfs_quota_groups="true"
+            btrfs_root_is_snapshot="true"
+            efipartsize="64"
+            editbootinstall="editbootinstall_rpi.sh"
+            bundle_format="%N-Image.%A-%I"
+        >
+            <systemdisk>
+                <volume name="home"/>
+                <volume name="root"/>
+                <volume name="tmp"/>
+                <volume name="opt"/>
+                <volume name="srv"/>
+                <volume name="boot/grub2/arm64-efi" mountpoint="boot/grub2/arm64-efi"/>
+                <volume name="usr/local"/>
+                <volume name="var" copy_on_write="false"/>
+            </systemdisk>
+            <oemconfig>
+              <oem-swap>true</oem-swap>
+              <oem-swapsize>1000</oem-swapsize>
+              <oem-skip-verify>true</oem-skip-verify>
+            </oemconfig>
+        </type>
+    </preferences>
+        <users>
+        <user password="$1$wYJUgpM5$RXMMeASDc035eX.NbYWFl0" home="/root" name="root" groups="root"/>
+    </users>
+          <repository type="rpm-md" alias="slebase">
+        <source path="http://updates.suse.de/SUSE/Products/SLE-Module-Basesystem/15-SP4/x86_64/product/"/>
+     </repository>
+      <repository type="rpm-md" alias="sle">
+        <source path="http://updates.suse.de/SUSE/Products/SLE-Product-SLES/15-SP4/x86_64/product/"/>
+     </repository>
+      <repository type="rpm-md" alias="sle-dev">
+        <source path="http://updates.suse.de/SUSE/Products/SLE-Module-Development-Tools/15-SP4/x86_64/product/"/>
+     </repository>
+     <repository type="rpm-md" >
+        <source path="http://updates.suse.de/SUSE/Products/SLE-Module-Server-Applications/15-SP4/x86_64/product/"/>
+     </repository>
```

Comment:
same as a bove

Reply:
I have modified the code and XML files to maintain generic across different SLEs versions.